### PR TITLE
Added debug logs + improved errors

### PR DESCRIPTION
**Pull request by guykeller**
## Status 
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19593

## Description
Added debug logs in relevant parts of several Archer commands, and improved the returned errors.

## Screenshots
Paste here any images that will help the reviewer

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Code Review

### CODE REVIEW

**Comment by Itay4** [comment]
@guykeller are we sure that all these debugs do not include sensitive data ?

File: Integrations/integration-ArcherRSA.yml [hunk 1]
```diff
@@ -2413,11 +2427,12 @@ script:
             try {
                 contentId = resp['RequestedObject']['Id'];
             } catch (err) {
-                throw "Archer didn't create a new record. We received the following response " + JSON.stringify(resp)
+                throw "Error: Archer returned an error and didn't create a new record.";
```
Comment (ronykoz) [inline Integrations/integration-ArcherRSA.yml]: why did you remove the real error?

File: Integrations/integration-ArcherRSA.yml [hunk 2]
```diff
@@ -2441,7 +2456,7 @@ script:
                     break;
                 } catch (err) {
                     if (i==14){
-                        throw "Archer didn't create a new record. We received the following response " + JSON.stringify(rawResponse);
+                        throw "Error: Archer returned an error and didn't create a new record. ";
```
Comment (ronykoz) [inline Integrations/integration-ArcherRSA.yml]: same here

File: Integrations/integration-ArcherRSA_CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
-Fixed the default field on which the search is performed.
+- Fixed the default field on which the search is performed.
```
Comment (ronykoz) [inline Integrations/integration-ArcherRSA_CHANGELOG.md]: didn't see that

**Comment by guykeller** [comment]
@Itay4 yes, I am sure

**Comment by guykeller** [comment]
Closing this as a new and fixed PR was opened
